fetch-subtree-tools

```diff
@@ -12,17 +12,17 @@ url=https://raw.githubusercontent.com/implicitconversions/git-subtree-fast/main
 # use bash exec to to do so, to avoid bash blowing up if the file changed.
 (( INSTALL_IS_SECOND_RUN || NO_AUTO_UPDATE )) || {
     echo "Performing self-update step..."
-    exec bash -c "curl $url/fetch-subtree-tools -o '$target_dir/fetch-subtree-tools' && chmod +x '$target_dir/fetch-subtree-tools' && export INSTALL_IS_SECOND_RUN=1 && '${BASH_SOURCE[0]}'"
+    exec bash -c "curl $url/fetch-subtree-tools -o '$target_dir/fetch-subtree-tools' && chmod +x '$target_dir/fetch-subtree-tools' && export INSTALL_IS_SECOND_RUN=1 && '${BASH_SOURCE[0]}' '${target_dir}'"
 }
 
 echo "Fetching scripts into $(realpath "$target_dir")"
 
 files_to_fetch=(
+    git-subtree-fast
```

Comment:
I considered replacing this with a find looking for `git-subtree*` but I'm not sure if that's overkill compared to just updating this to pull the correct filenames.

Reply:
find can't always work because the files don't exist when you install for the first time. It would have to enumerate files from the github remote via some complicated API, or by making a temporary clone and deleting it.

Just list them by hand, we don't expect this list to change much, if ever.